sys/uuid/uuid.c

```diff
@@ -39,7 +39,7 @@ void uuid_custom(void *buf, size_t len, int gen)
 {
     uuid_base(buf, len);
 
-    for (int i = 0; i < sizeof(gen); i++) {
+    for (size_t i = 0; i < sizeof(gen); i++) {
```

Comment:
there is another loop in this file that already uses `size_t`, hence I opted for consistency (file-wise). IIRC coding convention are a bit unclear about it, too.